pyntc/devices/asa_device.py

```diff
@@ -40,12 +40,11 @@ def __init__(self, host, username, password, secret="", port=22, **kwargs):
         self.open()
 
     def _enable(self):
-        self.native.exit_config_mode()
-        if not self.native.check_enable_mode():
-            self.native.enable()
+        print("This method will be deprecated; migrate to use the ``enable`` method")
```

Comment:
Nice! I like that this is wrapping the new public API while maintaining backwards-compatibility.

Preferable to use actual deprecation warnings here, though, so they can be controlled and filtered by the Application.

(Short summary with examples: https://www.programcreek.com/python/example/61499/warnings.DeprecationWarning)